### DISCUSSION TITLE
Sample app

### DISCUSSION THREAD

**Discussion by RuchaVs** [espressif/esp-iot-solution#321] [General]
1.How to use this app  given in this repository? i installed the app in my phone ,its showing my device's mac ID but when i click on my device ,it always shows not found any bins.
P.S:the required bootloader .bin, partition.bin and firmware to be updated file are in the same folder as that of app.

2.What changes should be made to sdk config file for ESP 32 S3 file if we want to use pre encrypted option mentioned in code.

**Comment by leeebo** [comment]
@RuchaVs Not really sure which topic are you talking about? which app?

**Reply by RuchaVs** [reply]
sample app mentioned in this . repo's readme in section 3
https://github.com/espressif/esp-iot-solution/tree/master/examples/bluetooth/ble_ota..

**Reply by lisir233** [reply]
for question 1: Please put the bin file into PhoneStorage: Android/data/com.espressif.bleota.android/files/BLE-OTA/
